src/api.js

```diff
@@ -12,6 +12,21 @@ export const fetchArticles =  () => {
 
 export const fetchArticleById = (id) => {
     return api.get(`/articles/${id}`)
+        .then((res) => res.data)
+        .catch((err) => {
+            console.log(err)
+            return err.response.data
+        });
+}
+
+export const fetchCommentsByArticleId = (id) => {
+    return api.get(`/articles/${id}/comments`)
+        .then((res) => res.data)
+        .catch((err) => console.log(err));
```

Comment:
Our catch blocks should be in our useEffect as that's the end of our promise chain 👍